docs/reference/filters.md

```diff
@@ -80,6 +80,23 @@ foo3:
   -> <dynamic>;
 ```
 
+
+## blockContent
+
+Block a request based on it's body content.
+
+The filter max request body is 2MiB by default and can be overidden with `-max-matcher-buffer-size=<int>`.
+
+Parameters:
+
+* toblockList (List of []bytes)
```

Comment:
It's a list of string instead of list of []byte from the user perspective, which is what we document here.